apps/payment/models.py

```diff
@@ -48,7 +52,7 @@ class Payment(models.Model):
     )
     """Which Stripe key to use for payments"""
 
-    payment_type = models.SmallIntegerField(_("type"), choices=TYPE_CHOICES)
+    payment_type = models.SmallIntegerField(_("type"), choices=PaymentTypes.choices)
```

Comment:
This requires a db migration (that does nothing). Run `./manage.py makemigrations` (needs to be formatted, but should be handled by the pre-commit hook if you have it set up)